internal/features/user_flags.go

```diff
@@ -31,7 +31,7 @@ type CognitiveAccountFeatures struct {
 type VirtualMachineFeatures struct {
 	DetachImplicitDataDiskOnDeletion bool
 	DeleteOSDiskOnDeletion           bool
-	GracefulShutdown                 bool
+	GracefulShutdown                 bool // TODO: Remove in 5.0
```

Comment:
We don't normally allow Major version TODOs in the code, but since this is unavoidable right now, I think a longer explanation should be present for whomever has to deal with this for 5.0?
```suggestion
	GracefulShutdown                 bool // TODO: Remove in 5.0 - Currently not possible to deprecate feature block struct items via feature flagging. Feature made redundant/ineffective by a breaking API change.
```

Reply:
Fixed.